Tests/BoardTest.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using TicTacToe;
+using Xunit;
+
+namespace Tests.TicTacToe {
+
+    public class BoardTest {
+
+        private Board board;
+        
+        public BoardTest() {
+            this.board = new Board();
+        }
+
+        [Theory]
+        [InlineData(0)]
+        [InlineData(4)]
+        public void AddMarkerToGameBoard(int move) {
```

Comment:
Even though there's only two cases, I like that you're aware of opportunities to use `Theory`.